juju/juju
# Peergrouper Watches Controller Config

## Description of change

When controller configuration is updated (in particular HA and management spaces), we want to wake up the peergrouper worker loop immediately.

## QA steps

- Configure MAAS to ensure that there are 3 nodes, each with addresses in 2 spaces common to all.
- Bootstrap with one of the spaces as HA space in controller config.
- Enable HA.
- Update the controller config HA space to the other common space.
- Ensure that everything is still working and that the Mongo rs.status() looks OK.

## Documentation changes

Ultimately, yes.

## Bug reference

N/A
